crates/core/src/remappings.rs

```diff
@@ -279,7 +283,7 @@ fn generate_remappings(
     }
 
     // sort the remappings
-    new_remappings.sort_unstable();
+    new_remappings.par_sort_unstable();
```

Comment:
Ok so I did a quick benchmark because i was curious. On this task of sorting a vec of strings, there is no clear benefit to using rayon, regardless of the Vec length (the strings are from a corpus of text and each contain 10 words)

![image](https://github.com/user-attachments/assets/51d1ef8e-3c18-49e5-b251-f8da21abdcfb)